target/linux/ath79/dts/ar9344_compex_wpj344-16m.dts

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	compatible = "compex,wpj344-16m", "qca,ar9344";
+	model = "Compex WPJ344 (16MB flash)";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status: status {
+			label = "wpj344:green:status";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		sig1 {
+			label = "wpj344:red:sig1";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		sig2 {
+			label = "wpj344:yellow:sig2";
+			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
+		};
+
+		sig3 {
+			label = "wpj344:green:sig3";
+			gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
+		};
+
+		sig4 {
+			label = "wpj344:green:sig4";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
```

Comment:
Please use gpio-keys and drop poll-interval.

Reply:
done - pushing in a moment